Makefile

```diff
@@ -244,5 +249,14 @@ run-container: ## Run container from specified IMAGE_TAG
 		$(CONTAINER_CLITOOL) run ${IMAGE_TAG}
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Dashboard Targets                                                                                               #
+# Example Dashboard Targets                                                                                       #
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+clean-example: clean ## Clean example packages and docker
```

Comment:
Can you rename the targets to something more specific like `clean-grafana-metrics-example`?